### PR TITLE
#3 Abbreviate push and done

### PR DESCRIPTION
**Pull request by fabriciofx**
As per #3.

### CODE REVIEW

File: git-work.sh [hunk 1]
```diff
@@ -31,47 +31,46 @@ usage() {
...
-
 case "$1" in
   # start a new branch
   issue)
+    [ -z $2 ] && ( usage && exit 1 )
     git checkout -b "$2"
     ;;
   # commit current work
   commit)
+    [ -z $2 ] && ( usage && exit 1 )
```
Comment (mdbs99) [inline git-work.sh]: I've got this error: `/git-work.sh: line 48: [: first: binary operator expected`

File: git-work.sh [hunk 2]
```diff
@@ -31,47 +31,46 @@ usage() {
...
     git commit -am "$2"
     ;;
   # finish current work and merge
   done)
-    git checkout master
-    git merge "$2"
+    branch=${2:-$(git symbolic-ref --short HEAD)}
```
Comment (mdbs99) [inline git-work.sh]: This is not right. See the log below.

It is supposed to do:
1. switch to `master`
2. merge in `master`

The command `git work done` didn't merge with `master`.

```bash
> git-work-test (master)
$ git work issue 1
Switched to a new branch '1'

> git-work-test (1)
$ git status
On branch 1
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   foo.txt

no changes added to commit (use "git add" and/or "git commit -a")
> git-work-test (1)
$ git add .

>git-work-test (1)
$ git work commit "changed"
[1 6ad1ec8] changed
 1 file changed, 2 insertions(+), 1 deletion(-)

> git-work-test (1)
$ git work done
Already up-to-date.
```

File: git-work.sh [hunk 3]
```diff
@@ -31,47 +31,46 @@ usage() {
...
   # push to the server
   push)
-    git push origin "$2"
+    branch=${2:-$(git symbolic-ref --short HEAD)}
+    git push origin "$branch"
     ;;
   # release a new tag
   release)
-    git checkout master
+    [ -z $2 ] && ( usage && exit 1 )
```
Comment (mdbs99) [inline git-work.sh]: I didn't test, but I believe that will be the same error https://github.com/mdbs99/git-work/pull/4/commits/5a27305364349f3f584563cf9343a3319e0c1443?diff=split#r131385035

Try to run both commands on `master`.

File: git-work.sh (hunk 1 above)
Comment (fabriciofx) [reply git-work.sh]: @mdbs99 Fixed! Thanks!

File: git-work.sh (hunk 2 above)
Comment (fabriciofx) [reply git-work.sh]: @mdbs99 Fixed! Thanks!

File: git-work.sh (hunk 3 above)
Comment (fabriciofx) [reply git-work.sh]: @mdbs99 Fixed! Thanks!

**Comment by fabriciofx** [comment]
@mdbs99 Could you check it now?

**Comment by mdbs99** [comment]
@fabriciofx it was merged. Thanks for the excellent job!

**Pull request merged**